Mage/src/main/java/mage/abilities/effects/common/CopyPermanentEffect.java

```diff
@@ -61,25 +65,64 @@ public CopyPermanentEffect(final CopyPermanentEffect effect) {
 
     @Override
     public boolean apply(Game game, Ability source) {
-        Player player = game.getPlayer(source.getControllerId());
-        MageObject sourceObject = game.getPermanentEntering(source.getSourceId());
-        if (sourceObject == null) {
-            sourceObject = game.getObject(source.getSourceId());
+        Player controller = game.getPlayer(source.getControllerId());
+        MageObject sourcePermanent = game.getPermanentEntering(source.getSourceId());
+        if (sourcePermanent == null) {
+            sourcePermanent = game.getObject(source.getSourceId());
         }
-        if (player != null && sourceObject != null) {
+        if (controller != null && sourcePermanent != null) {
             Permanent copyFromPermanent = null;
             if (useTargetOfAbility) {
                 copyFromPermanent = game.getPermanent(getTargetPointer().getFirst(game, source));
             } else {
                 Target target = new TargetPermanent(filter);
                 target.setNotTarget(true);
-                if (target.canChoose(source.getSourceId(), player.getId(), game)) {
-                    player.choose(Outcome.Copy, target, source.getSourceId(), game);
+                if (target.canChoose(source.getSourceId(), controller.getId(), game)) {
+                    controller.choose(Outcome.Copy, target, source.getSourceId(), game);
                     copyFromPermanent = game.getPermanent(target.getFirstTarget());
                 }
             }
             if (copyFromPermanent != null) {
-                bluePrintPermanent = game.copyPermanent(copyFromPermanent, sourceObject.getId(), source, applier);
+                bluePrintPermanent = game.copyPermanent(copyFromPermanent, sourcePermanent.getId(), source, applier);
+                
+                //if object is a copy of an aura, it needs to attach
+                if (bluePrintPermanent.hasSubtype(SubType.AURA, game)){
+                    //copied from mage.cards.c.CopyEnchantment.java
+                    Target target = bluePrintPermanent.getSpellAbility().getTargets().get(0);
+                    Outcome auraOutcome = Outcome.BoostCreature;
+                    for (Ability ability : bluePrintPermanent.getAbilities()) {
+                        if (ability instanceof SpellAbility) {
+                            for (Effect effect : ability.getEffects()) {
+                                if (effect instanceof AttachEffect) {
+                                    auraOutcome = effect.getOutcome();
+                                }
```

Comment:
You missing of `break Ability;` line. In old code auraOutcome selected by first effect, but now it's will be selected by last one.